cmd/generate/config/rules/atlassian.go

```diff
@@ -11,15 +11,23 @@ func Atlassian() *config.Rule {
 	r := config.Rule{
 		Description: "Detected an Atlassian API token, posing a threat to project management and collaboration tool security and data confidentiality.",
 		RuleID:      "atlassian-api-token",
-		Regex: utils.GenerateSemiGenericRegex([]string{
-			"atlassian", "confluence", "jira"}, utils.AlphaNumeric("24"), true),
+		Regex: utils.MergeRegexps(
+			utils.GenerateSemiGenericRegex(
+				[]string{"atlassian", "confluence", "jira"},
+				utils.AlphaNumeric("24"),
+				true,
+			),
+			utils.GenerateUniqueTokenRegex(`ATATT3[A-Za-z0-9_\-=]{186}`, false),
```

Comment:
Based on my testing, this can be more specific:
```suggestion
			utils.GenerateUniqueTokenRegex(`ATATT3[\w-]{177}=[A-F0-9]{8}`, false),
```